client/jetpack-cloud/sections/partner-portal/payment-methods/stored-credit-card-method.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CreditCardFields from 'calypso/jetpack-cloud/sections/partner-portal/credit-card-fields';
+import CreditCardPayButton from 'calypso/jetpack-cloud/sections/partner-portal/credit-card-fields/credit-card-pay-button';
```

Comment:
👋  Hi! This imports a file that doesn't exist. Was that intentional as part of an ongoing project, or is this abandoned? 

The reason I ask is that, since this file isn't TS, it's causing a type error in `client/jetpack-cloud/sections/partner-portal/payment-methods/hooks/use-create-stored-credit-card.ts` because it mistakenly thinks that the `activePayButtonText` prop must be undefined. In trying to fix that type error, I began converting this file to TS only to discover that it is invalid.

Reply:
Stripping out some of the non-existent imports in https://github.com/Automattic/wp-calypso/pull/59545

Reply:
Hi @sirbrillig. Kinda missed the ping somehow, but thank you for stripping out the unused code. That code was part of a feature team Infinity was working on by that time, but we had to focus on other high priority projects.
cc @Automattic/jetpack-infinity

Reply:
No problem! Feel free to re-add whatever code is needed when you get around to it!